extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -133,7 +133,6 @@ private synchronized void initializeProperties() {
                 if (vendor != null) {
                     updateProperty(Thing.PROPERTY_VENDOR, vendor);
                 }
-                updateProperty(LIGHT_ID, fullLight.getId());
```

Comment:
Please note that this change will break https://github.com/eclipse/smarthome/issues/3899 as long as https://github.com/eclipse/smarthome/issues/3863#issuecomment-320099215 isn't yet implemented (correct, @afuechsel?).